CONTRIBUTING.md

```diff
@@ -24,59 +24,96 @@
 
 This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
 By participating, you are expected to uphold this code.
-Please report unacceptable behavior to [security@libgd.org](mailto:security@libgd.org).
+Please report unacceptable behavior according to the [enforcement section](CODE_OF_CONDUCT.md#enforcement).
 
 ### Repositories
 
-* [The main libGD repository](https://github.com/libgd/libgd) for the library and utilities released as part of libgd
-* [The libgd website repository](https://github.com/libgd/website) for the libgd.org website
-* [The Pango support for libgd repository](https://github.com/libgd/gd-pango) for the for libGD Pango support
+* [The main libGD repository](https://github.com/libgd/libgd)
+  for the library and utilities released as part of libgd
+* [The libgd website repository](https://github.com/libgd/website)
```

Comment:
this has moved since :x.  it's now https://github.com/libgd/libgd.github.io.

Reply:
Yep, noticed that in the meantime. Also gd-pango should probably be archived; at least it doesn't make sense to refer to that ancient repo:

> More to come in the next weeks.
(2006-12-22)

I'll do a follow-up PR soon.